caffe2/serialize/inline_container.cc

```diff
@@ -190,6 +190,17 @@ bool PyTorchStreamReader::hasRecord(const std::string& name) {
   return result;
 }
 
+std::vector<std::string> PyTorchStreamReader::getAllRecords() {
+  mz_uint num_files = mz_zip_reader_get_num_files(ar_.get());
+  std::vector<std::string> out;
+  char buf[MZ_ZIP_MAX_ARCHIVE_FILENAME_SIZE];
```

Comment:
If you use `std::array<char, MZ_ZIP_MAX_ARCHIVE_FILENAME_SIZE>`, you can use `arr.size()` in `mz_zip_reader_get_filename` and avoid a future where one of them is updated but not the other.

Reply:
> where one of them is updated but not the other.


What do you mean? There's only 1 `MZ_ZIP_MAX_ARCHIVE_FILENAME_SIZE` symbol

Reply:
I mean during manual edits. The `size()` on `std::array` is constexpr so it's effectively identical to specifying the constant in line.